js/directives/wire-model.js

```diff
@@ -77,6 +82,22 @@ function getModifierTail(modifiers) {
         'lazy', 'defer'
     ].includes(i))
 
+    if (modifiers.includes('debounce')) {
```

Comment:
Please add a comment here to describe why we are removing it (because we're using livewire debounce and don't want Alpine debounce)